scripts/secnetperf.ps1

```diff
@@ -202,17 +202,15 @@ INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Kernel_mode, Run_arg
     for ($tcp = 0; $tcp -lt $Test.Values.Length; $tcp++) {
         if ($Test.Values[$tcp].Length -eq 0) { continue }
         $transport = $tcp -eq 1 ? "tcp" : "quic"
-        $json["$($Test.Metric)-$transport"] = $Test.Values[$tcp]
-        if ($Test.Metric.startsWith("throughput")) {
+        $json["$testId-$transport"] = $Test.Values[$tcp]
+        if ($Test.Metric -eq "throughput") {
```

Comment:
@ProjectsByJackHe please make sure you notice this. I am updating the json output to use the `testId` instead of the `metric` since metric is not unique. We will need to update the dashboard as well, right?